collage/layout.py

```diff
@@ -0,0 +1,213 @@
+    def open_text_window(self, event):
+        # Output: tk.canvas object
+        root = tk.Tk()
+        window = TextConfigureApp(master=root)
+        window.mainloop()
+        return window.get_return()
+        
+        
+class TextConfigureApp(tk.Frame):
+    def __init__(self, master=None):
+        super().__init__(master)
+        self.master = master
+        self.rgb = (0, 0, 0)
+        self.grid()
+        self.create_widgets()
+
+    def create_widgets(self):
+        
+        # TODO: sticky to window edges
+        
+        # Canvas with result
+        self.canvas = tk.Canvas(self, width = 100, height = 100)
+        self.canvas.grid(row = 0, column = 1, rowspan = 2)
+        
+        # Text redactor
+        text_frame = tk.Frame(self, bd = 10)
+        text_frame.grid(row = 0, column = 0)
+        label_text = tk.Label(text_frame, text = "Type text here: ", 
+                              padx = 10, pady = 10)
+        label_text.grid(row = 0, column = 0)
+        self.text_redactor = tk.Text(text_frame, width = 30, height = 10)
+        self.text_redactor.grid(row = 1, column = 0)
+        self.text_redactor.insert(tk.END, datetime.now().date().strftime("%B %Y"))
+        
+        # Font chooser
+        font_frame = tk.Frame(self, bd = 10)
+        font_frame.grid(row = 1, column = 0)
+        label_font = tk.Label(font_frame, text = "Select font: ", padx = 10, pady = 10)
+        label_font.grid(row = 2, column = 0)
+        self.font_box = tk.Listbox(font_frame, selectmode='SINGLE')
+        self.font_box.grid(row = 3, column = 0, rowspan = 3)
+        self.system_fonts = get_system_fonts()
+        for item in self.system_fonts:
+            self.font_box.insert(tk.END, item)
+        self.font_box.selection_set(0)
+        self.font_box.bind('<Double-Button-1>', self.choose_font)
+        buttons = tk.Frame(self, bd = 10)
+        buttons.grid(row = 1, column = 1)
+        
+        # TODO: add other text modifiers: italic, bold, underlined, font size
+        
+        # Color chooser
+        self.color_button = tk.Button(buttons, text = "Change color...", 
+                                      padx = 10, pady = 10, width = 10)
+        self.color_button.grid()
+        self.color_button.bind('<Button-1>', self.choose_color)
+        
+        # Try button
+        self.label_button = tk.Button(buttons, text = "Try font",
+                                      padx = 10, pady = 10, width = 10)
+        self.label_button.grid()
+        self.label_button.bind('<Button-1>', self.choose_font)
+        
+        # Add and quit button
+        self.ok_button = tk.Button(buttons, text = "OK",
+                                   padx = 10, pady = 10, width = 10)
+        self.ok_button.grid()
+        self.ok_button.bind('<Button-1>', self.quit)
+        
+    def choose_color(self, event):
+        self.rgb, self.color_name = askcolor(parent = self, title = "Choose color:")
+        print(self.rgb, self.color_name)
```

Comment:
I think it would be better for logging
https://docs.python.org/3/library/logging.html